src/api/java/baritone/api/utils/Rotation.java

```diff
@@ -80,6 +80,22 @@ public Rotation subtract(Rotation other) {
         );
     }
 
+    /**
+     * Returns a copy of the opposite rotation of the given rotation
+     *
+     * @return
```

Comment:
The CI build is failing on this. Maybe just put some documentation behind the `@return`?

Reply:
I couldn't find why it was failing... thanks!